sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -301,23 +306,22 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * Don't create too many partitions in parallel on a large cluster; otherwise Spark might crash
    * your external database systems.
    *
-   * @param url JDBC database url of the form `jdbc:subprotocol:subname`.
+   * You can find the JDBC-specific options for reading table via JDBC in
```

Comment:
Can we change: "JDBC-specific options for reading table" -> "JDBC-specific option and parameter documentation for reading tables"?